bin/jetify

```diff
@@ -9,14 +9,13 @@ rm -f $MAPPING_SED
 echo "Creating new sed command from $MAPPING_FILE"
 while IFS=, read -r from to
 do
-	echo "s/$from/$to/g;" >> $MAPPING_SED
+  echo "s/$from/$to/g;" >> $MAPPING_SED
 done <<< "$(cat $MAPPING_FILE)"
 echo "Command file for sed ready. Searching for files to jetify..."
 
 jetify() {
   echo "jetifying file $1"
-  sed -i".bak" -f $MAPPING_SED $FILE
-  rm -f "$FILE.bak"
+  sed -i -f $MAPPING_SED $FILE
```

Comment:
I like this idea and I was playing with it myself, but I think the file suffix for '-i' is mandatory on macOS sed - did you verify this works on macOS? My commit history looks clean but I must have bounced back and forth 6 times getting it to work the force time before I soft-reset and force-pushed it cleanly ;-)

Reply:
I thought you were going to sleep! :D 

It's funny you mention that, because the only documentation I've found about this behaviour said that OSX was the only system [that _supported_ it](https://stackoverflow.com/a/25486705/1701465)! Maybe it means we would have to declare the empty string for this platform? I have a Macbook lying around so I'll be able to pull this repo down and take a look.

Still trying to work on performance enhancements. Who needs sleep, anyway.